tasks/selinux_centos.yml

```diff
@@ -0,0 +1,10 @@
+- name: Set http_can_network_connect
+  seboolean:
+    name: httpd_can_network_connect
+    state: yes
+    persistent: yes
+
+# Disable SELinux
+- name: Disable SELinux (shouldn't be done!)
+  selinux:
+    state: disabled
```

Comment:
Regarding this problem, @dariomas suggested to set selinux in "permissive mode", see what kind of suggestions you get from selinux in its logs and add the necessary rules, do you think it's possible?